### DISCUSSION TITLE
Default ordering for podcasts

### DISCUSSION THREAD

**Discussion by jphastings** [advplyr/audiobookshelf-app#1123] [Q&A]
Is there a way to set the default order for a podcast to be oldest first?

I feel like there must be and I'm just an idiot for not spotting it!

(Every time an episode finishes everything stops (as there's nothing earlier left to play), and I have to open the app, reload the podcast's page, choose the sort order & wait for the UI to refresh before I can get to a subsequent episode; I'd love to remove this minor nuisance from my life!)

**Comment by AdrianEdelen** [comment]
+1. On the app I am replacing with abs, (pocketcasts) you can set filter and sort rules for each individual podcast. this is a great feature as some podcasts a user may just want the newest episodes always (news or daily episodes) and others they may want the oldest first (serials, multipart episodes)

**Reply by golgoth85** [reply]
+1, it shouldn't be hard to add and it's very useful when you manually import podcasts

**Comment by andonevris** [comment]
Development seems to have stalled on the app, there are loads of really essential QOL things that have been proposed for years but never get implemented.

I'm still waiting for the app to create a queue when something starts playing, the web player already does.  Coming to the end of a podcast or book when on the road is a PITA.

**Reply by nichwall** [reply]
The focus in recent months has been on the server and getting more of that cleaned up and taken care of. Once that is done, there will be more focus on the mobile apps.
